README.md

```diff
@@ -107,11 +114,18 @@ The second container is a bit smaller if the size is important to you.
 
 # Philosophy
 
-Nu draws inspiration from projects like PowerShell, functional programming languages, and modern CLI tools. Rather than thinking of files and services as raw streams of text, Nu looks at each input as something with structure. For example, when you list the contents of a directory, what you get back is a table of rows, where each row represents an item in that directory. These values can be piped through a series of steps, in a series of commands called a 'pipeline'.
+Nu draws inspiration from projects like PowerShell, functional programming languages, and modern CLI tools.
+Rather than thinking of files and services as raw streams of text, Nu looks at each input as something with structure.
```

Comment:
Is it better in markdown to put each line on its own line?  I haven't seen that before

Reply:
Most markdown renderers ignore line breaks and wrap sequential lines into a paragraph of prose. Paragraphs are typically separated by blank lines. That's how the spec defines it, both in the original markdown definition (section [Paragraphs and line breaks](https://daringfireball.net/projects/markdown/syntax#p)), and in the CommonMark spec (section [Soft line breaks](https://spec.commonmark.org/0.29/#soft-line-breaks)).

(For some reason, GitHub renders single line breaks as hard breaks in comments like this, but not in actual source code like README.md files.)

As for why it's better, I'll refer to the oft-quoted post [Semantic linefeeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/), which explains the reasoning better than I could.

Reply:
That does make sense, thanks!